app/src/main/java/ar/com/wolox/android/training/ui/errors/ErrorHandler.java

```diff
@@ -0,0 +1,16 @@
+package ar.com.wolox.android.training.ui.errors;
+
+public class ErrorHandler {
+    public static String getErrorMessage(ErrorCode errorCode) {
+        switch (errorCode) {
+            case EMPTY_FIELDS:
+                return "Todos los campos son obligatorios";
```

Comment:
Try to get the text from the resources.

Reply:
Its done later (https://github.com/wolox-training/dm-android/blob/home-page-3/app/src/main/java/ar/com/wolox/android/training/ui/errors/ErrorHandler.java)